db/seeds.rb

```diff
@@ -6,7 +6,9 @@
 
 organization = Organization.create_or_find_by(name: '100eyes',
                                               upgrade_discount: 10,
-                                              business_plan: BusinessPlan.first)
+                                              business_plan: BusinessPlan.where.not(
```

Comment:
`BusinessPlan` looks like a model that I don't know of yet, because I review not often enough.